src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -17,7 +17,6 @@
 use PHPCR\PropertyType;
 use PHPCR\SessionInterface;
 use Sulu\Bundle\ContentBundle\Document\PageDocument;
-use Sulu\Bundle\TagBundle\Entity\Tag;
```

Comment:
that's not really possible. because we would have to set this config value (this would break other tests). The only way would be to unit test the NodeController which would lead into a mocking hell currently.